python/tank/util/shotgun.py

```diff
@@ -223,8 +223,16 @@ def __create_sg_connection(config_data=None, user=None):
         sg = shotgun_api3.Shotgun(
             config_data["host"],
             script_name=config_data["api_script"], api_key=config_data["api_key"],
-            http_proxy=config_data.get("http_proxy")
-        )
+            http_proxy=config_data.get("http_proxy"), connect=config_data.get("connect", True))
```

Comment:
Would we need to force connect to ```False``` if ```config_data``` has ```timeout_secs``` in it?  Or is that redundant with the code below?